### ISSUE TITLE
Check Responsiveness / Accessibility

### ISSUE THREAD

**Issue by gingerhale** [gingerhale/cpnt260-project-board#3]
## Description
Content should be responsive to fit on any screen, also be "viewable" for screen readers

## Definition of Done
- [x] Site has no horizontal scrolling
- [x] Line length is a max of 65 characters
- [x] Content is centred
- [x] Content is at least 1rem away from edge of screen
- [x] Wrapped in main element
- [x] Contrast ratio exceeds AA
